cam_intrins: [1709.163406, 0.000000, 622.365608, 
0.000000, 1710.142290, 510.444884, 
0.000000, 0.000000, 1.000000]

lidar_points: [-0.684974,-0.120213,2.59368,
-0.469293,0.110623,2.57841,
-0.685159,0.34766,2.56708,
-0.923678,0.122768,2.61067,
-0.270257,-0.0468155,2.61481,
-0.004215,0.155576,2.58391,
-0.211281,0.410428,2.60212,
-0.436304,0.184721,2.5651]

cam_points: [-0.220728,-0.171364,2.32042,
-0.00868795,0.0477898,2.32641,
-0.227788,0.259327,2.3429,
-0.439828,0.0401736,2.33692,
0.189015,-0.0949976,2.31796,
0.41139,0.106031,2.26173,
0.209075,0.327372,2.25294,
-0.0132997,0.126343,2.30917,
]

lidar_to_cam: [  0.994886,-0.0298675, 0.0964886,  0.198841,
 0.0301409,  0.999545, -0.001377,-0.0464631,
-0.0964036,0.00427821,  0.995333, -0.313835,
         0,          0,          0,          1]

